tests/playbooks/kubevirt_raw_vm.yml

```diff
@@ -29,30 +29,23 @@
                      memory: 512M
                  devices:
                    disks:
-                   - volumeName: myvolume
-                     name: mydisk
-                     disk:
+                   - disk:
                        bus: virtio
-                   - name: cloudinitdisk
-                     volumeName: cloudinitvolume
-                     cdrom:
+                     name: bootdisk
+                     volumeName: bootvolume
+                   - cdrom:
                        bus: virtio
+                     name: cloudinitdisk
+                     volumeName: cloudinitvolume
                volumes:
-               - name: myvolume
+               - name: bootvolume
                  registryDisk:
                    image: kubevirt/fedora-cloud-registry-disk-demo:latest
-               - cloudInitNoCloud:
+               - name: cloudinitvolume
+                 cloudInitNoCloud:
                    userData: |
                      #cloud-config
-                     hostname: test-fedora
-                     users:
-                       - name: kubevirt
-                         gecos: KubeVirt Project
-                         sudo: ALL=(ALL) NOPASSWD:ALL
-                         passwd: $6$JXbc3063IJir.e5h$ypMlYScNMlUtvQ8Il1ldZi/mat7wXTiRioGx6TQmJjTVMandKqr.jJfe99.QckyfH/JJ.OdvLb5/OrCa8ftLr.
-                         shell: /bin/bash
-                         home: /home/kubevirt
-                         lock_passwd: false
-                         ssh_authorized_keys:
-                             - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC+KMp1F4SrKitvgILf8Q2GQbcOslIg4sfqUOf+m832zwoS2Z2ndmpr/3F+NtLq56cTaEzK9IPBIkNLJfsPqRsToLBYZD+l4S5FfuZ2xwCxGBH+TVXApO+SiD6c84rmjOx47665iQvMHKL+n/5gVvSdYDuegNKnj4rRr/eHnG2yC4TVZl3oHI7TPOUJT+kKjSWP1UesWTZmmck39IaFSmorg31X7g9hJHwq9JEDQilcbnIyqDZKiH6Ju4GjOU8mqhazBFB4qu/QDDb25pDpPd2pQGBilGvm7gwJCVnyDk9YZQU7gYE734KLDf5tCKMmEQSjwFx2Tj9mfZveCIJkaj3T kubevirt
-                 name: cloudinitvolume
+                     password: fedora
+                     chpasswd: { expire: False }
+                     ssh_authorized_keys:
+                       - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDizcn50tdTi4TiaUP3tuyLKQSgXmAMXkF/s8pNb+2agcudaxtwsz+2I9MH8Ty9eXDTMg3b5kqzNQRtuNyG8QX3Lm+Q0M2rTIPVxe6ZkVbdgXzSPQ0Lx8GUBu/AF0qv40sC2Rmbu1vbc2ogSHwkj/yDnGPoMNiVeG3Z6ZGFRwuLznU+C0owILOGgf6m83gNhALomLmNxsov+y+vUh5ofyEW0ve21htpsM3G/ui4/8hXVK8P92By84jzNkTCv6yTIPbpZ3XUr9AGTt5YgfGHMBAoyJ1rEsjjroDeGn0pVhIxeqsTP1/ej//hKwDvpMlbcrsWN6lAg0qfv4ikNf7OE7/N test@kubevirt
```

Comment:
Why not use here the ` tests/test_rsa.pub`?

Reply:
You mean lookup('file', …)? Good point, I'll change it in a sec.